pylmod/membership.py

```diff
@@ -20,5 +20,142 @@ def __init__(
             urlbase='https://learning-modules.mit.edu:8443/',
     ):
         super(Membership, self).__init__(cert, urlbase)
+
         # Add service base
         self.urlbase += 'service/membership/'
+
+    def get_staff(self, gradebook_id=''):
+        """Get the staff roster for a gradebook.
+
+        Get a list of staff members for a given gradebook,
+        specified by a gradebookid.
+
+        Args:
+            gradebook_id (str): unique identifier for gradebook, i.e. ``2314``
+
+        Raises:
+            requests.RequestException: Exception connection error
+            ValueError: Unable to decode response content
+
+        Returns:
+            list: list of dictionaries containing staff data
+
+            An example return value is:
+
+            .. code-block:: python
+
+                {
+                    u'data':
+                    {
+                        u'COURSE_ADMIN':
```

Comment:
Because this is structured upside down like sections, we should provide a method to reverse it.  Also a "simple" argument that changes the return to just a list of `accountEmails`